util/stmtsummary/statement_summary.go

```diff
@@ -235,11 +238,25 @@ type StmtExecInfo struct {
 // newStmtSummaryByDigestMap creates an empty stmtSummaryByDigestMap.
 func newStmtSummaryByDigestMap() *stmtSummaryByDigestMap {
 	sysVars := newSysVars()
+
+	now := time.Now().Unix()
+	intervalSeconds := sysVars.getVariable(typeRefreshInterval)
+	beginTimeForCurrentInterval := now - now%intervalSeconds
```

Comment:
Why do we need to initialize this variable? It seem useless.

Reply:
you are right, its time record updates automatically on every insertion.